### PR TITLE
feat: write react error log and update config

### PR DESCRIPTION
**Pull request by daniel-yh**
*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md#commit-your-change)
- [x] I have updated any necessary documentation, including READMEs and comments (where appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have checked that my tests are not configured for a specific environment

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react/src/components/CustomerContribution/CustomerFeedbackModal.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ export const CustomerFeedbackModal: React.FC<Props> = React.memo(({ visible, set
                       })
                     );
                   } else {
-                    console.error("Failed to send customer feedback: ", response.errorValue);
+                    window.electron.writeReactErrLog("Failed to send customer feedback - PA UI: " + response.errorValue)
```
Comment (aniruddhadave) [inline packages/react/src/components/CustomerContribution/CustomerFeedbackModal.tsx:85]: Using this format to write the logs will help the lambda use it further:
https://github.com/aws/porting-assistant-dotnet-ui/blob/33c64aaf78d89612bb0d1c443c2a40659a17a81d/packages/csharp/PortingAssistant/PortingAssistant.Api/Program.cs#L34
You can check in the above link how its is logged.

File: packages/react/src/components/CustomerContribution/PackageRuleContribution.tsx [hunk 2]
```diff
@@ -115,6 +115,7 @@ const PackageRuleContributionInternal: React.FC<Props> = ({ source }) => {
         });
         history.push(nextPagePath);
       } else {
+        window.electron.writeReactErrLog("Failed to send rule contribution - PA UI: " + result.errorValue)
```
Comment (aniruddhadave) [inline packages/react/src/components/CustomerContribution/PackageRuleContribution.tsx:118]: Same, use the standard format.

File: packages/electron/src/telemetry/electron-telemetry.ts [hunk 3]
```diff
@@ -95,6 +95,16 @@ export const logReactMetrics = (response: any) => {
   reactLogger.info(JSON.stringify(errorMetric));
 };
 
+export const logReactError = (response: any) => {
+  const cur = new Date();
+  let curTimeArr = cur.toISOString().split('T');
+  let curDate = curTimeArr[0];
+  let curTime = curTimeArr[1].split('.')[0];
+  let appVersion = app.getVersion();
+  let errorMessage = `[${curDate} ${curTime} ERR] (${appVersion}) ${response}`
```
Comment (aniruddhadave) [inline packages/electron/src/telemetry/electron-telemetry.ts:104]: Use this format 
https://github.com/aws/porting-assistant-dotnet-ui/blob/33c64aaf78d89612bb0d1c443c2a40659a17a81d/packages/csharp/PortingAssistant/PortingAssistant.Api/Program.cs#L34

Comment (daniel-yh) [reply packages/electron/src/telemetry/electron-telemetry.ts]: Yes, I updated codes. Thanks!

**Review by aniruddhadave — APPROVED** [review]

**Pull request merged**